docs/sources/reference/api/docker_remote_api_v1.12.md

```diff
@@ -115,7 +115,8 @@ Create a container
              "Memory":0,
              "MemorySwap":0,
              "CpuShares": 512,
-             "Cpuset": "0,1",
+             "CpusetCpus": "0,1",
```

Comment:
Not sure that it is good idea to change docs for previous versions. ping @SvenDowideit